junit-platform-launcher/src/testFixtures/java/org/junit/platform/launcher/core/NamespacedHierarchicalStoreProviders.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2015-2025 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.launcher.core;
+
+import static org.junit.platform.engine.support.store.NamespacedHierarchicalStore.CloseAction.closeAutoCloseables;
+
+import org.junit.platform.engine.support.store.Namespace;
+import org.junit.platform.engine.support.store.NamespacedHierarchicalStore;
+
+public class NamespacedHierarchicalStoreProviders {
+
+	public static NamespacedHierarchicalStore<Namespace> dummyNamespacedHierarchicalStore() {
+		return new NamespacedHierarchicalStore<>(new NamespacedHierarchicalStore<>(null), closeAutoCloseables());
```

Comment:
The parent store should also use `closeAutoCloseables()`.

```suggestion
		return new NamespacedHierarchicalStore<>(dummyNamespacedHierarchicalStoreWithNoParent(), closeAutoCloseables());
```